style.css

```diff
@@ -0,0 +1,455 @@
+html, body {
+	width: 100%;
+	height: 100%;
+}
+
+html {
+	font: 15px 'montserratregular';
+}
+
+body {
+	margin: 0;
+	position: relative;
+	min-width: 320px;
+	max-width: 100%;
+	align-items: center;
+}
+
+h1,h2 {
+	margin: 0;
+	padding: 0;
+}
+a {
+	font-family: 'montserratsemibold';
+	text-decoration: none;
+	color: black;
+}
+
+button {
+	border: none;
+	background: none;
+	cursor: pointer;
+	outline: none;
+}
+
+header {
+display: flex;
+flex-wrap: nowrap;
+justify-content: space-between;	
+}
+.header__pic {
+  margin: 10px;
+}
+
+.menu {
+	margin: 30px;
+	padding: 0;
+	display: flex;
+	flex-wrap: nowrap;
+justify-content: center;	
+}
+
+.header__aside {
+   display: flex;
+	margin: 30px 30px;
+}
+
+.header__menu-link {
+	margin: 20px;
+	padding: 0;
+	color: black;
+
+}
+
+.header__menu-link:hover {
+	color: #E51937;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)